templates/base.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html>
+  <head>
+    {% block head %}
+  <link rel="stylesheet" href="static/styles.css">
+  <title>{% block title %} Seats held by women {% endblock %}</title>
+    {% endblock %}
+  </head>
+  <body>
+
+<div class="grid-container">
+<header class="header"></header>
+<main class="main">{% block content %}{% endblock %}</main>
+
+<aside class="sidenav">
+  <p>Select country</p>
+
+  <form method="POST" action="/">
+
+
+  <select name="country" class='dropbtn'>
+
+  <option value="hide">-- country --</option>
+  {% for country in table['country'].unique() %}
+          <option value="{{ country }}">{{ country }}</option>
+      {% endfor %}
```

Comment:
I think the indentation is off here - the `endfor` part should have the same as the `for`